### ISSUE TITLE
Extension version and download not working in Ublue Bazzite/Bluefin

### ISSUE THREAD

**Issue by LiberalTugboat** [celiopy/auto-adwaita-colors#10]
On new installs of Ublue Bazzite and Bluefin (based on Fedora Silverblue) the download button does not function and even after installing the icons manually the extension says they are Not yet Installed. However, the icons do change to match the accent color as expected.

Manually installed using this command (after creating the folder because it is not there by default):
For immutable distros like Fedora Silverblue, the global installation is different, use:
sudo cp -r ./Adwaita-colors/* /var/usrlocal/share/icons/
![Screenshot From 2024-12-11 13-18-25](https://github.com/user-attachments/assets/a1873058-d741-4985-8476-be358337c2cf)

**Comment by celiopy** [comment]
The extension itself is still working right? The color syncs with system?

The extension as it is now only cares and works with local files, not system files. That's why it says it isn't installed.

Next version will see a lot of changes, including this behavior, and prefs screen will be completed redone.

The idea is for it to realize that the icons are installed somewhere else and completely disable updating features.

We will also support injecting accent colors to gtk3 apps if adw-gtk3 is installed.

**Comment by LiberalTugboat** [comment]
Yes, the extension itself is working and the colors sync with accent color changes.
